src/api/spec/controllers/person_controller_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+# WARNING: If you change tests make sure you uncomment this line
+# and start a test backend. Some of the actions
+# require real backend answers for projects/packages.
+# CONFIG['global_write_through'] = true
+
+RSpec.describe PersonController, vcr: false do
+  describe 'POST #post_userinfo' do
+    let(:user) { create(:confirmed_user) }
+
+    before do
+      login user
+    end
+
+    context 'when in LDAP mode' do
+      before do
+        stub_const('CONFIG', CONFIG.merge({ 'ldap_mode' => :on }))
+      end
+
+      it 'user is not allowed to change their password' do
+        post :post_userinfo, params: { login: user.login, cmd: 'change_password', format: :xml }
```

Comment:
[nitpick] You could also move it to the before block to make the test more atomar